swagger/api.yaml

```
@@ -72,6 +72,45 @@ paths:
   "/enterprise/v3/enterprise-customer/{uuid}/learner-summary":
     $ref: "https://raw.githubusercontent.com/edx/edx-enterprise-data/eb793cf/api.yaml#/endpoints/v2/enterpriseCustomerLearnerSummary"
 
+  # Added by TNL team to support Studio content API
+  "api/contentstore/v1/file_asset/{proxy+}":
+    x-amazon-apigateway-any-method:
+      parameters:
+      - name: "proxy"
+        in: "path"
+      x-amazon-apigateway-integration:
+        type: "http_proxy"
+        uri: "https://${stageVariables.studio_host}/api/contentstore/v1/file_asset/{proxy}"
+        httpMethod: "ANY"
+        requestParameters:
+          integration.request.path.proxy: "method.request.path.proxy"
+
+  # Added by TNL team to support Studio content API
+  "api/contentstore/v1/video_asset/{proxy+}":
```

This is not correct, in neither of your routes. It explains why the tests don't work.
```suggestion
  "/contentstore/v1/video_asset/{proxy+}":
```

---

I double-checked both parts of your assertion against the registrar routes (i.e., that we need for a forward slash, and that it's an error to include the leading "api") , and the registrar code is consistent with your points. The routes do include a leading /api, but the yaml file does not. Moreover, when I made and pushed the changes you suggested, the tests passed. Thanks!!!